src/MeetingsSDKAdapter.test.js

```diff
@@ -87,33 +101,65 @@ describe('Meetings SDK Adapter', () => {
       mockSDKMeeting.getMediaStreams = jest.fn((constraint) => Promise.resolve([constraint.sendAudio ? ['localAudio'] : ['localVideo']]));
     });
 
-    test('throws errors and nullifies local Audio if not retrieved successfully', async () => {
-      mockSDKMeeting.getMediaStreams = jest.fn(() => Promise.reject());
-      global.console.error = jest.fn();
-      const localAudio = await meetingSDKAdapter.getStream(meetingID, {sendAudio: true});
+    test('throws errors, nullifies local Audio and sets permissions to denied, dismissed or error if not retrieved successfully', (done) => {
```

Comment:
actually, it doesn't throw errors, just logs them

maybe: "logs errors and returns a null stream and error status if stream cannot be retrieved"